Source/WebCore/page/DeprecatedGlobalSettings.h

```diff
@@ -131,11 +131,6 @@ class DeprecatedGlobalSettings {
     static bool opusDecoderEnabled() { return shared().m_opusDecoderEnabled; }
 #endif
 
-#if ENABLE(MEDIA_SOURCE) && (HAVE(AVSAMPLEBUFFERVIDEOOUTPUT) || USE(GSTREAMER))
```

Comment:
Ditto

Reply:
there's zero reference to MediaSourceInlinePainting settings in GStreamer code.